.drone.env

```diff
@@ -1,7 +1,7 @@
 # The test runner source for API tests
-CORE_COMMITID=90edbb40800b36fadc4aba6216a3fd16b559f1e3
-CORE_BRANCH=master
+CORE_COMMITID=36177b6ae2304e6462d2c76554278691721db521
+CORE_BRANCH=refactor-scenario
```

Comment:
Why did you change the branch name?

Reply:
Depends on https://github.com/owncloud/core/pull/38493. I will change it to master after it is merged.

Reply:
oh! ok